src/SearchResults.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import moment from "moment";
+import CustomerProfile from "./CustomerProfile";
+
+const SearchResult = props => {
+  const [selectedId, setSelectedId] = useState();
+
+  const setSelected = id => {
+    setSelectedId(id);
+  };
+
+  return (
+    <div>
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="col">Id</th>
+            <th scope="col">Title</th>
+            <th scope="col">First Name</th>
+            <th scope="col">Surname</th>
+            <th scope="col">Email</th>
+            <th scope="col">Room Id</th>
+            <th scope="col">Check In Date</th>
+            <th scope="col">Check Out Date</th>
+            <th scope="col">Nights</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.results.map((item, index) => (
+            <TableRow item={item} index={index} setSelected2={setSelected} />
+          ))}
+        </tbody>
+      </table>
+      <CustomerProfile id={selectedId} />
+    </div>
+  );
+};
+
+const TableRow = props => {
+  const [selected, setSelected] = useState(false);
+
+  const clickHandler = () => {
+    setSelected(!selected);
+  };
+
+  const handleCustomerId = id => {
```

Comment:
I also think you don't need this function, i.e. you can do this
```
<button
     className="btn btn-primary"
     onClick={() => {
        props.setSelected2(props.item.id);
     }}
>
```

Reply:
Thank you Ellie, it makes sense, I changed it .